README.md

```diff
@@ -80,8 +82,18 @@ This extension provides 4 actions:
     ```php
     <?php
 
+    namespace app\controllers;
+
+    use Yii;
+    use yii\web\Controller;
+    use yii\helpers\Url;
+
+
     class SamlController extends Controller {
 
+        // Remove CSRF protection
+        public $enableCsrfValidation = false;
```

Comment:
do we need to remove CSRF protection for login action? i think it is ok if CSRF protection still enabled for login action. but for ACS action we need to disable CSRF.